keps/2936-local-queue-defaulting/README.md

```diff
@@ -0,0 +1,63 @@
+# KEP-2936: LocalQueue defaulting
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1](#story-1)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+<!-- /toc -->
+
+## Summary
+
+This KEP introduces a defaulting LocalQueue mechanism for the jobs that don't have
+LocalQueue name annotation specified.
+
+## Motivation
+
+Simplify user experience with Kueue.
+
+### Goals
+
+- default LocalQueue for jobs that don't specify LocalQueue name annotation.
+
+### Non-Goals
+
+- create a default LocalQueue.
+
+## Proposal
+
+If the Job doesn't have annotation for LocalQueue in its Spec and the LocalQueue
+with name `default` is present in Job's namespace and KueueDefaulting feature gate
```

Comment:
Does it only apply if there is only one LQ in namespace named `default` or there can be more